extensions/sources/sql/ext/sql.go

```diff
@@ -41,6 +42,9 @@ type sqlsource struct {
 	Query sqlgen.SqlQueryGenerator
 	// The db connection instance
 	db *sql.DB
+
+	RuleID     string
```

Comment:
Actually, these information are already in the context. Is it possible not to add another ref? I don't think the SQL source buisiness logic relates to the rule name, stream name.

Reply:
updated.